02_bubble/bubble.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(s []int) []int {
+	n := len(s)
+	swapped := true
+	for swapped {
+		swapped = false
+		for i := 1; i < n; i++ {
+			if s[i-1] > s[i] {
+				s[i], s[i-1] = s[i-1], s[i]
+				swapped = true
+			}
+		}
+	}
+	return s
```

Comment:
Is mutating the input slice permitted in the problem statement?

Reply:
Not sure if the problem statement talks about mutating